jps-plugin/src/com/reason/StringUtil.java

```diff
@@ -15,4 +16,15 @@ public static String toFirstUpper(@Nullable String value) {
 
     return value.substring(0, 1).toUpperCase(Locale.getDefault()) + value.substring(1);
   }
+
+  public static @Nullable String trimLastCR(@Nullable String value) {
+    if (value == null || value.isEmpty()) {
```

Comment:
`StringUtils.isBlank(value)`

Reply:
I'm trying to stay away from apache.commons